examples/textual_inversion/textual_inversion.py

```diff
@@ -930,7 +940,15 @@ def main():
                 commit_message="End of training",
                 ignore_patterns=["step_*", "epoch_*"],
             )
-
+    elapsed_time = time.time() - start_time
+    throughput = (len(train_dataset)*args.num_train_epochs) / elapsed_time
```

Comment:
Should be much better if it shows throughput more frequently using something like log_interval option during the training process unless such more frequent throughput calculation makes the whole process too slow.

Reply:
I will implement this in next PR